src/Generator/AbstractTypeGenerator.php

```diff
@@ -62,13 +62,19 @@ function (%s) <closureUseStatements>{
      */
     private $expressionLanguage;
 
+    /**
+     * @var int
+     */
+    protected $cacheDirMask;
```

Comment:
Since the value can be set using construct making this private in a first place could ease maintenance.
```suggestion
    private $cacheDirMask;
```